inst/extdata/packages/kuenm.yaml

```diff
@@ -0,0 +1,48 @@
+name: kuenm
+title: Detailed Development of Ecological Niche Models Using Maxent
+version: 1.1.10
+author: "Marlon E. Cobos, A. Townsend Peterson, Luis Osorio-Olvera,
+Narayani Barve"
+maintainer: Marlon E. Cobos <manubio13@gmail.com>
+cran: no
+repository: https://github.com/marlonecobos/kuenm
+description: "A set of tools to help with the development of critical phases of the
+ecological niche modeling process in Maxent. Pre-modeling analyses and
+explorations can be done to prepare data. Model calibration (model selection)
+can be done by creating and testing several candidate models. Handy options
+for producing final models, evaluating such models, and assessing extrapolation
+risks are also included. Tools for post-modeling analyses are implemented to
+allow for further exploration of results. Methodological and theoretical b
+bases for the methods implemented here can be found in:
+Peterson et al. (2011) <https://www.degruyter.com/princetonup/view/title/506966>,
+Barve et al. (2011) <doi:10.1016/j.ecolmodel.2011.02.011>,
+Owens et al. (2013) <doi:10.1016/j.ecolmodel.2013.04.011>,
+Radosavljevic and Anderson (2014) <doi:10.1111/jbi.12227>,
+Peterson et al. (2028) <doi:10.1111/nyas.13873>,
+Cobos et al. (2019a) <doi:10.7717/peerj.6281>,
+Cobos et al. (2019b) <doi:10.1016/j.ecoinf.2019.100983>,
+Cobos et al. (2019c) <doi:10.1101/603100>."
+occ_acquisition: no
+occ_cleaning: yes
```

Comment:
Not clear what function "exploring records outside layer borders included" refers to. If it means to_closest, I suppose that can qualify as cleaning.